types/fastify-static/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for fastify-static 0.14
+// Project: https://github.com/fastify/fastify-static
+// Definitions by: Leonhard Melzer <https://github.com/leomelzer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import { Server, IncomingMessage, ServerResponse } from "http";
+import * as fastify from "fastify";
```

Comment:
Why `import *` here but `import = require` in the tests?

Reply:
Thanks for the review! I've changed it to `require` to align both versions.